tools/universe/package.py

```diff
@@ -0,0 +1,34 @@
+from .package_manager import PackageManager
+from .version_resolver import VersionResolver
+
+
+class Package:
+
+    def __init__(self, name, version, version_resolver=None):
+        self._name = name
+        self._version = version
+
+        self._set_version_resolver(version_resolver)
```

Comment:
Having this broken out in a separate function feels a little overkill, would it make sense to have the contained logic directly under `__init__`?